CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
+
+- **OpenAI Tracing** - We've enhanced our OpenAI integration with a new tracing feature that works seamlessly with MLflow OpenAI autologging. Users can now enable tracing of their OpenAI API usage with a single `mlflow.openai.autolog()` call, then MLflow automatically logs valuable metadata such as token usage, providing deeper insights into your OpenAI-powered applications. To start exploring this new capability, please check out [the tracing documentation](https://mlflow.org/docs/latest/llms/tracing/index.html#automatic-tracing)! (#12267, @gabrielfu)
+
+- **Enhanced Model Deployment with New Validation Feature** - To improve the reliability of model deployments, MLflow has added a new method to validate your model before deploying it to the inference endpoint. This feature helps eliminate typical errors in input and output handling, streamlining the process of model deployment and increasing confidence in your deployed models. By catching potential issues early, you can ensure a smoother transition from development to production. (#12710, @serena-ruan)
+
+- **Custom Metrics Definition Recording for Evaluations** - We've strengthened the flexibility of defining custom metrics for model evaluation by automatically logging and versioning metrics definitions, including models used as judges and prompt templates. With this new capability, you can ensure reproducibility of evaluations across different runs and easily reuse evaluation setups for consistency, facilitating more meaningful comparisons between different models or versions. (#12487, #12509, @xq-yin)
+
+- **Databricks SDK Integration** - MLflow's interaction with Databricks endpoints has been fully migrated to use the [Databricks SDK](https://docs.databricks.com/en/dev-tools/sdk-python.html), replacing the legacy CLI. This change brings more robust and reliable connections between MLflow and Databricks, and access to the latest Databricks features and capabilities. (#12313, @WeichenXu123)
+
+
+### Other Notable Changes
+
+Features:
+
+- [Tracking] Make parent id a parameter (#12721, @Flametaa)
+- [Tracking] Make MLflow supporting U2M authentication (#12713, @WeichenXu123)
+- [Models] Add more chat params for ChatModel (#12612, @WeichenXu123)
+- [Models] [Transformers] Support input images encoded with b64.encodebytes (#12087, @MadhuM02)
+- [Models Registry] Handle AWS encryption details (#12495, @artjen)
+
+Bug fixes:
+
```

Comment:
Can similar changes as above be applied to these descriptions below here?

Reply:
Sure!